packages/jest-config/src/resolveConfigPath.ts

```diff
@@ -95,3 +119,15 @@ const makeResolutionErrorMessage = (
   `traverse directory tree up, until it finds one of those files in exact order: ${JEST_CONFIG_EXT_ORDER.map(
     ext => `"${getConfigFilename(ext)}"`,
   ).join(' or ')}.`;
+
+const makeMultipleConfigsError = (configPaths: Array<Config.Path>) =>
+  [
+    '● Multiple configurations found:',
+    ...configPaths.map(configPath => `  * \`${configPath}\``),
```

Comment:
I don't think we need the backticks here - terminal output is monospace anyways. Beyond that I like this! 🙂

Reply:
✅